events/hfe_events_population.txt

```diff
@@ -1020,7 +1020,7 @@ country_event = {
 				base = 1
 				modifier = {
 					add = 10
-					# проверяется отношение империи
```

Comment:
I see not meaningful changes in this file

Reply:
See my comment on the first issue about the removal of the comments.